packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -107,6 +108,19 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
             return this.streamFile(file, res, { range, overrideHeaders: { "Cache-control": "private" } });
         }
 
+        @PublicApi()
+        @Get(`/download/${fileUrl}`)
+        async downloadFile(@Param() { fileId }: FileParams, @Res() res: Response, @Headers("range") range?: string): Promise<void> {
```

Comment:
comparing to hasedFileUrl:

- why does this use PublicApi and not DisableGlobalGuard?
- why is cdnOriginCheck not needed?
- why don't we need a hash?

Reply:
@thomasdax98 should I use DisableGlobalGuard here instead of PublicApi?
Is the cdnOriginCheck here necessary?

Reply:
If you don't have any reason for changes, this should be a 1:1 copy of hasedFileUrl with the additional header